app/AppKernel.php

```diff
@@ -375,6 +419,6 @@ protected function unlockCacheStream($lockStream): void
      */
     public function getAppType(): string
     {
-        return $this instanceof \AdminKernel ? 'admin' : 'front';
+        return $this instanceof \FrontKernel ? 'front' : 'admin';
     }
```

Comment:
You don't check the new kernel?

Reply:
Both OAuthApiKernel and AdminKernel are considered "admin" for now, so it was simpler to check the only one that isn't